less/_components/block.less

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2020 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Martin Lowe <martin.lowe@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+*/
+
+.eclipsefdn-block {
```

Comment:
I would move this to the classes-blocks.less file. I don't think we need a separate file for this component.